torch/csrc/autograd/profiler.cpp

```diff
@@ -139,6 +142,13 @@ struct ProfilerThreadLocalState
       auto& list = kv.second;
       result.emplace_back(list->consolidate());
     }
+    // Consolidate remote events if applicable as well.
+    if (remoteProfiledEvents_) {
+      result.insert(
```

Comment:
ok, so all remote events will be in the same vector? I guess that's fine, just wanted to double check

Reply:
Yes, this portion is putting them into the `result` vector that all thread events and remote events will be in after `disableProfiler`. Currently `remoteProfiledEvents_` will contain events from all remote nodes over the profiler's range, which seems fine to me for now - we can separate them out if there is a use case in the future.